app/api/helpers/custom_fields.py

```diff
@@ -27,8 +27,8 @@ def format(self, value):
         works only for GET requests
         """
         if not self.validate(value):
-            print 'Validation of field with value \"%s\" (%s) failed' % (
-                value, str(self.__class__.__name__))
+            #print 'Validation of field with value \"%s\" (%s) failed' % (
```

Comment:
I commented this out because in the comment says "disabling for development purposes as the server crashes when exception is raised. can be enabled when the project is mature". So I thought maybe we could look into this later. If you say I will remove completely.